### DISCUSSION TITLE
GitHub actions not getting triggered today?

### DISCUSSION THREAD

**Discussion by oledid** [community/community#25886] [Actions]
<p>I was testing actions just fine yesterday, but today my actions won’t start?</p>
<p><a href="https://github.com/oledid/oledid.SyntaxImprovement/actions" rel="nofollow noopener">https://github.com/oledid/oledid.SyntaxImprovement/actions</a></p>

**Comment by chasdevs** [comment]
<p>I’m seeing the same thing, granted I just upgraded to the new syntax and have not seen them successfully run yet…</p>

**Comment by oledid** [comment, marked as answer]
<p>I was able to trigger a run today, but it failed:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/341bf38db9af501df0dee817e7f1ac9fd282e842.png" data-download-href="/uploads/short-url/7qYIgHlfatFBHCcdxIa09eyihJ8.png?dl=1" title="Capture.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1911049/181093822-21571690-392f-4064-bccb-7ce0113e2a1d.png" alt="Capture.PNG" data-base62-sha1="7qYIgHlfatFBHCcdxIa09eyihJ8" width="690" height="310" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/341bf38db9af501df0dee817e7f1ac9fd282e842_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.PNG</span><span class="informations">926×417 16.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So I guess this might have been the problem all along?</p>
